v2/validator.go

```diff
@@ -21,16 +21,18 @@ import (
 var val *validator.Validate
 
 const (
-	autoEventFrequencyTag       = "edgex-dto-autoevent-frequency"
+	dtoFrequencyTag             = "edgex-dto-frequency"
```

Comment:
This change impacts AutoEvent DTO correct. Updates needed there?

Reply:
Since the AutoEvent and Interval both have a frequency field, so I rename the tag to make it more general for reusing.

Reply:
I understand and agree with the name change, but expected to see changes to the AutoEvent DTO to use new name.

Reply:
No problem, change made there https://github.com/edgexfoundry/go-mod-core-contracts/pull/427/files#diff-fbe92df0cf3b16ad6c0f28c3528270b3310d83530f06fe639ca331a0096ece02R14

Reply:
Thanks!